samples/MvcCoreApp/Program.cs

```diff
@@ -21,7 +21,9 @@
         .Configure(options =>
         {
             options.RemoteAppUrl = new(builder.Configuration["ReverseProxy:Clusters:fallbackCluster:Destinations:fallbackApp:Address"]);
-            options.ApiKey = ClassLibrary.RemoteServiceUtils.ApiKey;
+
+            // Do not re-use this API key. Every solution should have a unique API key.
```

Comment:
do we need this here? the code itself is fine to copy

Reply:
Moved to config